appveyor.yml

```diff
@@ -0,0 +1,35 @@
+build: false
+shallow_clone: false
+skip_branch_with_pr: true
+environment:
+
+  global:
+    PYTHONUNBUFFERED: "1"
+    INSTALL_EDM_VERSION: "1.9.1"
```

Comment:
Is there any reason not to have the latest EDM here?

Reply:
I remember there being issues with the existing `install-edm-windows.cmd` script and the latest edm version 1.11.0. I'll investigate this and get back to you.

Reply:
yay! fixed the `install-edm-windows.cmd` script and we now use edm 1.11.0